content/docs/user-guide/external-dependencies.md

```diff
@@ -220,6 +203,10 @@ Importing 'model.pkl (git@github.com:iterative/example-get-started)'
 The command above creates `model.pkl.dvc`, where the external dependency is
 specified (with the `repo` field).
 
+⚠️ DVC won't push or pull imported data to
```

Comment:
feels excessive to explain the details in every example. That's what descriptions for to my mind

Reply:
This doc is a guide about external deps, there's no description per se, just a **How it works** section and I don't think this note would make any sense there. Importing is only mentioned in the examples.

Reply:
But I guess we don't really need these notes in this doc, as remote storage is also not the topic, and they're in the import/-url cmd refs. Removing.